web/libs/editor/src/tags/object/TimeSeries/MultiChannel.jsx

```diff
@@ -0,0 +1,95 @@
+import { observer } from "mobx-react";
+import { types } from "mobx-state-tree";
+import Types from "../../../core/Types";
+import { TagParentMixin } from "../../../mixins/TagParentMixin";
+import Registry from "../../../core/Registry";
+import TimeSeriesVisualizer from "../../../components/TimeSeries/TimeSeriesVisualizer";
+import ChannelLegend from "./ChannelLegend";
+import { getChannelColor } from "./palette";
+
+// MultiChannel model for grouped display of channels
+const Model = types
+  .model("MultiChannelModel", {
+    id: types.optional(types.identifier, () => Math.random().toString(36).slice(2)),
+    type: "multichannel",
+    children: Types.unionArray(["channel"]),
+    parentTypes: Types.tagsTypes(["TimeSeries"]),
+    height: types.optional(types.string, "200"),
+
+    showaxis: types.optional(types.boolean, true),
```

Comment:
shouldn't it be consistent?
```suggestion
    showxaxis: types.optional(types.boolean, true),
```

Reply:
nope. it's for both.